server/logical.sql

```diff
@@ -0,0 +1,68 @@
+DROP TABLE IF EXISTS users CASCADE; 
+DROP TABLE IF EXISTS cohort CASCADE; 
+DROP TABLE IF EXISTS events CASCADE; 
+DROP TABLE IF EXISTS recipes CASCADE; 
+DROP TABLE IF EXISTS dietary_restrictions CASCADE; 
+DROP TABLE IF EXISTS allergies CASCADE; 
+DROP TABLE IF EXISTS recipe_allergies CASCADE; 
+
+CREATE TABLE users(
+    id SERIAL PRIMARY KEY,
+    user_name VARCHAR(60) NOT NULL,
+    user_email VARCHAR(90) NOT NULL,
```

Comment:
Email address can be up to 254 characters. It's best not to try and assume things about emails at all, just restrict to the bare minimum.

Reply:
Thanks for the input, its really helpful.